### ISSUE TITLE
Version mismatch

### ISSUE THREAD

**Issue by WalrusInAnus** [Gribbleshnibit8/xcom2-launcher#3]
When I start the program, I get a popup telling me the current version is 0.4.4 and new one 1.0.0, but that's not what GitHub says.

**Comment by Gribbleshnibit8** [comment]
This will be fixed in the next version. Finally figured out how the version thing works.

**Comment by velit** [comment]
This bug still exists as of the newest version, which the updater thinks is 1.0.2 even though version on github says 1.0.1 and so does the release zip.

**Comment by Littlebob86** [comment]
![untitled](https://cloud.githubusercontent.com/assets/25976048/23660474/811c412e-0384-11e7-84bd-d7e60ff662c0.jpg)

**Comment by Gribbleshnibit8** [comment]
Yep, sure does. I really don't understand the version thing (the project that it's from has documentation that basically says not to use it how it's being used here. I will probably just disable it and do something else because I just don't know...

**Comment by Littlebob86** [comment]
So am I ok to reassign all my mods, or are you likely to update the launcher again and I lose all my categorised mods?? :)

**Comment by BTernaryTau** [comment]
Backup your settings.json file if you want to be sure you don't lose your categories.

**Comment by Littlebob86** [comment]
Much appreciated :)



Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by kwstannard** [comment]
I took a crack at tracking this down. So, the current version number is gotten from XCom2Launcher.GitVersion*Information*. I can't find this defined anywhere. There is, however, a GitVersion*Config*.yaml, but the values within do not seem to be used anywhere.

Have you tried explicitly setting "v1.0.1" as the return value of GetCurrentVersion and seeing if that fixes the bug? I am guessing that is probably not ideal, but I don't know how to use C#.

What this doesn't answer is where the value "v1.0.2" comes from. I can't find this reference in the code anywhere and doesn't come from the github json information either.

**Comment by ghost** [comment]
FYI guys, this was forked and current launcher maint. is below

https://github.com/X2CommunityCore/xcom2-launcher
